project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
-addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.0.6")
+addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.0.7")
 addSbtPlugin("com.dwijnand"       % "sbt-dynver"               % "4.0.0")
-addSbtPlugin("com.jsuereth"       % "sbt-pgp"                  % "1.1.2")
+addSbtPlugin("com.jsuereth"       % "sbt-pgp"                  % "2.0.0")
```

Comment:
We had some issues with version 2.0.0 in ZIO which made releases fail. Could you revert just that one? Until we find a workaround.

Reply:
@takayahilton i saw that you fixed the other issue (thanks!), maybe you missed this one? We need to stay with sbt-pgp 1.1.2 otherwise there will be errors during releases :)